### PR TITLE
Refactor router isis

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

refactor `eos_cli_config_gen/templates/eos/router-isis.j2`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes)
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2 [hunk 1]
```diff
@@ -1,23 +1,33 @@
 {# eos - router isis #}
-{% if router_isis is defined and router_isis is not none %}
+{% if router_isis is arista.avd.defined %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2]: change this to `router_isis.instance` and skip the next one

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2 [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
    !
-{%     if router_isis.address_family is defined and router_isis.address_family is iterable %}
-{%          for address_family in router_isis.address_family %}
+{%     if router_isis.address_family is arista.avd.defined %}
+{%         for address_family in router_isis.address_family %}
    address-family {{ address_family }}
-{%             for af_default in router_isis.isis_af_defaults %}
-      {{af_default}}
-{%             endfor %}
+{%             if  router_isis.isis_af_defaults is arista.avd.defined %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2]: two spaces after if :)

**Review by ClausHolbechArista — COMMENTED** [review]
Two minor findings.

**Review by ClausHolbechArista — APPROVED** [review]

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2 (hunk 1 above)
Comment (titom73) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2]: Fix with commit 965ff23

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2 (hunk 2 above)
Comment (titom73) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2]: Fix with commit 965ff23

**Pull request merged**